web-app/src/services/holidayService.js

```diff
@@ -2,18 +2,16 @@ import axios from '../utilities/AxiosInstance';
 
 export const cancelHoliday = holidayId => {
   return axios.post(`/holidays/cancel/${holidayId}`);
-}
+};
 
 export const getHolidays = employeeId => {
   return axios.get(`/holidays/findByEmployeeId/${employeeId}`);
 };
 
-export const requestHoliday = holiday => {
-  return axios.post('/holidays/', holiday);
-};
-
-export const requestHolidays = holidays => {
-  return axios.post('/holidays/createMultiple/', holidays);
+export const requestHoliday = holidays => {
+  return holidays.length === 1 ?
+    axios.post('/holidays/', holidays[0]) :
+    axios.post('/holidays/createMultiple/', holidays);
```

Comment:
I don't think it makes sense for two calls for this, passing an array by default up to the server makes more sense. This shouldn't be a big change on both sides.